### ISSUE TITLE
Formatting track changes

### ISSUE THREAD

**Issue by kuldeep-k** [NYTimes/ice#117]
Hi,

I need to set track changes of formatting ( Bold, Italic etc. ) . But current plugin do not provide any feature.  I forked the repository [https://github.com/kuldeep-k/ice/tree/formatting] and do some changes but they do not work when i select html already having some start/end tags like <p> In this case they show html in tinymce. 
If it can be fixed or anybody do this before?

**Comment by jeevani2it** [comment]
hi,

you got it ?

**Comment by kuldeep-k** [comment]
No, Last year I checked that no solution possible. Changing library requires lot of works and understanding. Also HTML DOM Fragment is regular bottleneck to achieve solution :(

**Comment by jeevani2it** [comment]
Oh. So it is not possible. The only way is to customize the core files ?

**Comment by jeevani2it** [comment]
what you used alternate for this ?

**Comment by kuldeep-k** [comment]
I tell my company to situation and obviously they did not grant me unlimited time for R&D. Also I think customize core may not work as when I tried to develop custom code I faced same HTML DOM Fragment issue. So may not possible by using HTML DOM. However using pure string / regex operations may help but that would gone out of library scope.

**Comment by kuldeep-k** [comment]
I tell my company to situation and obviously they did not grant me unlimited time for R&D. Also I think customize core may not work as when I tried to develop custom code I faced same HTML DOM Fragment issue. So may not possible by using HTML DOM. However using pure string / regex operations may help but that would gone out of library scope.
